apollo-router/src/services/router.rs

```diff
@@ -196,6 +202,7 @@ impl Response {
         self.response.body_mut().next().await
     }
 
+    #[deprecated]
```

Comment:
I'm happy to deprecate this, but I wonder what the motivation is.

Reply:
I think that was to remove again the `Body` type from the public API. We could instead expose a stream of `Bytes` to map over